pkg/server/internal/packetio.go

```diff
@@ -150,19 +150,21 @@ func (p *PacketIO) readOnePacket() ([]byte, error) {
 		if _, err := io.ReadFull(r, header[:]); err != nil {
 			return nil, errors.Trace(err)
 		}
+
+		// To be compatible with MariaDB Java Connecter 2.X, 
```

Comment:
```suggestion
		// To be compatible with MariaDB Connector/J 2.x, 
```

This seems to be the product name according to https://mariadb.com/kb/en/about-mariadb-connector-j/